lang/php7-pecl-imagick/Makefile

```diff
@@ -0,0 +1,33 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PECL_NAME:=imagick
+PECL_LONGNAME:=Provides a wrapper to the ImageMagick library
```

Comment:
I'd suggest "Image Processing (ImageMagick binding)", it's a little bit shorter.